README.md

```diff
@@ -138,6 +138,18 @@ structured, and other resources useful to developers new to RISC Zero, see our
 | risc0-zkvm               | [![x](https://img.shields.io/badge/crates.io-v0.14-orange)](https://crates.io/crates/risc0-zkvm)               | [![](https://img.shields.io/docsrs/risc0-zkvm)](https://docs.rs/risc0-zkvm)                             |
 | risc0-zkvm-platform      | [![x](https://img.shields.io/badge/crates.io-v0.14-orange)](https://crates.io/crates/risc0-zkvm-platform)      | [![](https://img.shields.io/docsrs/risc0-zkvm-platform)](https://docs.rs/risc0-zkvm-platform)           |
 
+## Feature flags
+
+The following feature flags are present in one or more of the crates listed above:
+
+| Feature | Target(s)  | Implies | Description | Crates |
+| --- | --- | --- | ---- | --- |
+| cuda |windows, linux| prove, std | Turns on CUDA GPU acceleration for the prover | rv32im, zkp, zkvm |
```

Comment:
I think the crate names need to be the full names, not shorthands

Reply:
Oh, good call -- perhaps ideally even docs.rs links? Full name feels more definitely important, though.